### PR TITLE
7251 chg registration confirmation msg

### PR DESCRIPTION
**Pull request by AlexKutkanych**

### CODE REVIEW

File: conference/react-js/src/containers/SignUpForm/SignUpForm.jsx [hunk 1]
```diff
@@ -46,6 +47,18 @@ class SignUp extends Component {
           })
         ));
     };
+
+    this.checkPassLen = () => {
```
Comment (ScreamCat1) [inline conference/react-js/src/containers/SignUpForm/SignUpForm.jsx]: Remove this. please

File: conference/react-js/src/containers/SignUpForm/SignUpPopUp/SignUpPopUp.jsx [hunk 2]
```diff
@@ -1,26 +1,17 @@
...
       <h3 className="pop-up__title">
         You&#39;ve successfully registered.
       </h3>
-      <p className="pop-up__notification">
-        <span className="pop-up__notification-user">{props.userName}</span>
-          ,&ensp;go to&ensp;
-        <span className="pop-up__notification-user">{props.mail}</span>
-    to complete the sign-up process</p>
+      <p className="pop-up__notification pop-up__notification_small">
+        Please check your email to complete the registration.</p>
```
Comment (ScreamCat1) [inline conference/react-js/src/containers/SignUpForm/SignUpPopUp/SignUpPopUp.jsx]: I think, close tag "p" should move to new line

File: conference/react-js/src/scss/_common.scss [hunk 3]
```diff
@@ -199,10 +199,13 @@ html {
 
     &-user {
       word-wrap: break-word;
-      font-family: "Oswald Bold";
+      font-family: "Oswald Bold", sans-serif;
       display: inline-block;
       max-width: 100%;
       }
+
+    &_small {}
```
Comment (ScreamCat1) [inline conference/react-js/src/scss/_common.scss]: What do this selector?

Comment (AlexKutkanych) [reply conference/react-js/src/scss/_common.scss]: add margin to confirmation message.
moved margin from row 208 to this class)

File: conference/react-js/src/containers/SignUpForm/SignUpForm.jsx [hunk 4]
```diff
@@ -15,27 +15,29 @@ class SignUp extends Component {
       mail: '',
       password: '',
       confirm: '',
+      passLen: false,
```
Comment (SupraVlad) [inline conference/react-js/src/containers/SignUpForm/SignUpForm.jsx]: not critical, but for future, use full self-descriptive names.

File: conference/react-js/src/containers/SignUpForm/SignUpForm.jsx [hunk 5]
```diff
@@ -15,27 +15,29 @@ class SignUp extends Component {
...
+    formChangeHandler = (e) => {
       const target = e.target;
 
       this.setState({
         [target.name]: target.value,
       });
     };
 
-    const registration = ({ mail, password, confirm, fname, lname }) => {
+    registration = ({ mail, password, confirm, fname, lname }) => {
```
Comment (SupraVlad) [inline conference/react-js/src/containers/SignUpForm/SignUpForm.jsx:33]: the same here. `fname` and `lname` may be unclear for sb

**Review by SupraVlad — APPROVED** [review]

File: conference/react-js/src/containers/SignUpForm/SignUpForm.jsx (hunk 4 above)
Comment (AlexKutkanych) [reply conference/react-js/src/containers/SignUpForm/SignUpForm.jsx]: ok, that's mine, I'll change it)

File: conference/react-js/src/containers/SignUpForm/SignUpForm.jsx (hunk 5 above)
Comment (AlexKutkanych) [reply conference/react-js/src/containers/SignUpForm/SignUpForm.jsx:33]: lname and fname are those we need to send to backend in the body of our request (same listed on wiki)

**Review by asanametove — APPROVED** [review]

**Pull request merged**